hudi-common/src/main/java/org/apache/hudi/common/fs/FSUtils.java

```diff
@@ -80,9 +80,6 @@
   private static final PathFilter ALLOW_ALL_FILTER = file -> true;
 
   public static Configuration prepareHadoopConf(Configuration conf) {
-    conf.set("fs.hdfs.impl", org.apache.hadoop.hdfs.DistributedFileSystem.class.getName());
```

Comment:
this is some old old code. Have you ensured that this can run on a non HDFS filesystem now? (HDFS is testing in unit and integration tests already)

Reply:
@vinothchandar  On my machine, the implementation of filesyetem is `org.apache.hadoop.fs.viewfs.ViewFileSystem`. IMO, following user's actual configuration will be bertter.

Reply:
Just to be clear. This code basically says, if you use `hdfs:` or `file:` schemes, these classes are used. Of course, we honor the users configuration. otherwise we wont be able to write to S3 or GCS :)

Reply:
I wanna express that this hard code will ignore users configuration.